airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -52,14 +43,40 @@ def __init__(
         http_path: Optional[str] = None,
         sql_endpoint_name: Optional[str] = None,
         session_configuration: Optional[Dict[str, str]] = None,
+        http_headers: Optional[List[Tuple[str, str]]] = None,
+        catalog: Optional[str] = None,
+        schema: Optional[str] = None,
+        **kwargs,
     ) -> None:
+        """
+        Initializes DatabricksSqlHook
```

Comment:
Please move documentarion for init, because it is not visible in reference doc. See: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html by default, option autoclass_content is set to class.

Reply:
ah, sorry, mixed up the doc style with other project.